neural_lam/utils.py

```diff
@@ -233,11 +235,18 @@ def fractional_plot_bundle(fraction):
     return bundle
 
 
-def init_wandb_metrics(wandb_logger, val_steps):
+def init_training_logger_metrics(training_logger, val_steps):
     """
-    Set up wandb metrics to track
+    Set up logger metrics to track
     """
-    experiment = wandb_logger.experiment
-    experiment.define_metric("val_mean_loss", summary="min")
-    for step in val_steps:
-        experiment.define_metric(f"val_loss_unroll{step}", summary="min")
+    experiment = training_logger.experiment
+    if isinstance(training_logger, WandbLogger):
+        experiment.define_metric("val_mean_loss", summary="min")
+        for step in val_steps:
+            experiment.define_metric(f"val_loss_unroll{step}", summary="min")
+    elif isinstance(training_logger, MLFlowLogger):
+        pass
+    else:
+        warnings.warn(
+            "Only WandbLogger & MLFlowLogger is supported for tracking metrics"
+        )
```

Comment:
I guess what happens in this case is that experiment results will only go to stdout? It would be good to clarify that in this warning.